src/common/form/index.js

```diff
@@ -3,22 +3,20 @@ import Icon from 'icon'
 import map from 'lodash/map'
 import randomPassword from 'random-password'
 import React from 'react'
-import VirtualizedSelect from './virtualized-select'
-import Component from './base-component'
+
+import Component from '../base-component'
 import {
   autobind,
   formatSizeRaw,
   parseSize,
   propTypes
-} from './utils'
+} from '../utils'
 import {
   DropdownButton,
   MenuItem
 } from 'react-bootstrap-4/lib'
 
-// ===================================================================
-
-export { VirtualizedSelect as Select }
+export Select from './select'
```

Comment:
Maybe it does :) ([source](https://github.com/leebyron/ecmascript-more-export-from#proposed-additions))